includes/admin/upgrades/upgrade-functions.php

```diff
@@ -150,6 +153,16 @@ function give_show_upgrade_notices( $give_updates ) {
 			'callback' => 'give_v1813_update_donor_user_roles_callback',
 		)
 	);
+
+	// v1.8.14 Upgrades for donations.
+	$give_updates->register(
```

Comment:
Very few of our customers are using Iranian currency - can we only run the update if the base currency is set to rial? Otherwise it's a needless burden on 99% of our customers.

Reply:
Yes, it makes sense, the upgrade will not show if base currency is not RIAL. :)